terraform/core/38-api-ingestion.tf

```
@@ -115,8 +115,8 @@ module "copy_vonage_data_landing_to_raw" {
   trigger_enabled            = local.is_production_environment
   job_parameters = {
     "--job-bookmark-option" = "job-bookmark-enable"
-    "--raw_zone_bucket"    = "${module.raw_zone.bucket_id}/"
-    "--landing_zone_bucket"    = "${module.landing_zone.bucket_id}/"
+    "--raw_zone_bucket"    = "${module.raw_zone.bucket_id}"
+    "--landing_zone_bucket"    = "${module.landing_zone.bucket_id}"
```

If you're not creating a string you just need them like this:

```suggestion
    "--raw_zone_bucket"    = module.raw_zone.bucket_id
    "--landing_zone_bucket"    = module.landing_zone.bucket_id
```